ansible/roles/ci/doc/setup/semaphore.md

```diff
@@ -0,0 +1,29 @@
+# Semaphore Server
```

Comment:
our approaches are partially described in file best_practices.md. Regarding Slack is up to people to select notification approaches. In semaphore there is nice UI for that.